components/org.wso2.carbon.identity.sso.saml/pom.xml

```diff
@@ -198,29 +354,8 @@
                             org.apache.xml.security.c14n ; version="[1.4.2.patched,2.0.0]",
                             org.apache.xml.security.signature ; version="[1.4.2.patched,2.0.0]",
                             org.apache.xml.security.utils ; version="[1.4.2.patched,2.0.0]",
-                            org.opensaml,
-                            org.opensaml.common,
-                            org.opensaml.common.impl,
-                            org.opensaml.saml1.core,
-                            org.opensaml.saml2.core,
-                            org.opensaml.saml2.core.impl,
-                            org.opensaml.saml2.encryption,
-                            org.opensaml.ws.security,
-                            org.opensaml.ws.transport.http,
-                            org.opensaml.xml,
-                            org.opensaml.xml.encryption,
-                            org.opensaml.xml.io,
-                            org.opensaml.xml.schema,
-                            org.opensaml.xml.schema.impl,
-                            org.opensaml.xml.security,
-                            org.opensaml.xml.security.credential,
-                            org.opensaml.xml.security.criteria,
-                            org.opensaml.xml.security.keyinfo,
-                            org.opensaml.xml.security.x509,
-                            org.opensaml.xml.signature,
-                            org.opensaml.xml.signature.impl,
-                            org.opensaml.xml.util,
-                            org.opensaml.xml.validation,
+                            org.opensaml.*,
+                            net.shibboleth.utilities.java.support.*,
```

Comment:
Check if we need to remove the star imports and replace with the relevant imports

Reply:
This is fine